src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -149,7 +149,7 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             href: {
-                condition: (el) => {
+                condition: el => {
```

Comment:
I wonder if latest eslint has linting options for this. The older version haven't. But AFAIK there was a proposal for this. Can you take a look, please and add eslint rule in context of this PR?